tests/NotificationTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+use App\User;
+use Carbon\Carbon;
+use App\Conference;
+use App\Notifications\CFPIsOpen;
+use App\Events\ConferenceCreated;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Notification;
+use App\Listeners\SendNotificationForOpenCFPs;
+
+class NotificationTest extends IntegrationTestCase
+{
+    /** @test */
+    function if_approved_conference_is_created_users_are_notified()
+    {
+        Notification::fake();
+        factory(App\User::class)->states('enabledNotifications')->create();
+        $conference = factory(Conference::class)->create(['approved' => true]);
+
+        $listener = new SendNotificationForOpenCFPs();
+        $listener->handle(new ConferenceCreated($conference));
+
+        $conference = Conference::first();
+
+        Notification::assertSentTo(User::EnabledNotifications()->get(), CFPIsOpen::class, function ($notification) use ($conference) {
```

Comment:
maybe instead assert it was sent to $user and then track $user in line 18 so we're not testing the scope here